File: Java_React_Ödevler/Java_React_Ödev_Gün_04_Ödev_03_OyunSatis/Gun_04_Odev_03/Concrete/CampaignManager.java

public class CampaignManager extends BaseCampaignManager {

	public void add(Campaign campaign)
    {
        super.add(campaign);
        System.out.println("Kampanya kaydý eklendi" + " " + campaign.getId() + " " + campaign.getName());
    }
	
    public void update(Campaign campaign)
    {
    	super.update(campaign);
    	System.out.println("Kampanya kaydý güncellendi" + " " + campaign.getId() + " " + campaign.getName());
    }
    
    public void delete(Campaign campaign)
    {
    	super.delete(campaign);
        System.out.println("Kampanya kaydý silindi" + " " + campaign.getId() + " " + campaign.getName());
        
    }
}

File: KampIntro/KampIntro_Odevler/Gun_05_Odev_05/Gun_05_Odev_05/Concrete/CampaignManager.cs
using Gun_05_Odev_05.Abstract;
using Gun_05_Odev_05.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Gun_05_Odev_05.Concrete
{
    public class CampaignManager : BaseCampaignManager
    {
        public override void Add(Campaign campaign)
        {
            base.Add(campaign);
        }
        public override void Update(Campaign campaign)
        {
            base.Update(campaign);
        }
        public override void Delete(Campaign campaign)
        {
            base.Delete(campaign);
            Console.WriteLine("Kampanya kaydı silindi");
            Console.ReadLine();
        }
    }
}
